docs/ref/contrib/gis/db-api.txt

```diff
@@ -220,13 +220,14 @@ For example, let's say we have a ``SouthTexasCity`` model (from the
 in southern Texas::
 
     from django.contrib.gis.db import models
+    from django.db import models as default_models
 
     class SouthTexasCity(models.Model):
         name = models.CharField(max_length=30)
         # A projected coordinate system (only valid for South Texas!)
         # is used, units are in meters.
         point = models.PointField(srid=32140)
-        objects = models.GeoManager()
+        objects = default_models.Manager()
```

Comment:
`django.contrib.gis.db.models.Manager` should do since `django.db.models.*` is imported there.